alert.css

```diff
@@ -0,0 +1,68 @@
+.alert {
+  border-radius: 0.5rem;
+  font-weight: bold;
+  margin: 1rem 0;
+  max-width: 25rem;
```

Comment:
I think for alerts width would be better in percentage like 70% or 80%

Reply:
doned the changes max-widht: 70%;